pipeline/alert.go

```diff
@@ -733,6 +737,65 @@ type HipChatHandler struct {
 	Token string
 }
 
+// Send the alert to JIRA.
+//
+// Example:
+//    [jira]
+//      enabled = true
+//      url = ""
+//      username = ""
+//      password = ""
+//      project = ""
+//      issue_type = ""
+//      priority_warn = ""
+//      priority_crit = ""
+//      global = false
+//
+// Example:
+//    stream
+//         |alert()
+//             .jira()
+//
+//
+// If the 'jira' section in the configuration has the option: global = true
+// then all alerts are sent to JIRA without the need to explicitly state it
+// in the TICKscript.
+//
+// Example:
+//    [jira]
+//      enabled = true
+//      url = "https://your.jira.host"
+//      username = "test-username"
+//      password = "test-password"
+//      project = "TEST"
+//      issue_type = "Bug"
+//      priority_warn = "Normal"
+//      priority_crit = "Urgent"
+//      global = true
+//
+// Example:
+//    stream
+//         |alert()
+//             .jira()
+//
+// Send alert to JIRA using project 'TEST'.
+// tick:property
+
+func (a *AlertNode) Jira() *JiraHandler {
+	jira := &JiraHandler{
+		AlertNode: a,
+	}
+	a.JiraHandlers = append(a.JiraHandlers, jira)
+	return jira
+}
+
+// tick:embedded:AlertNode.Jira
+type JiraHandler struct {
+	*AlertNode
+
+	Project string
```

Comment:
I think the issue_type, priority_warn, and priority_crit should all be configurable per alert, and if its empty then use the value from the config.